src/main/kotlin/app/revanced/patches/youtube/layout/widesearchbar/fingerprints/WideSearchbarTwoParentFingerprint.kt

```diff
@@ -5,6 +5,6 @@ import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
 import org.jf.dexlib2.AccessFlags
 
 object WideSearchbarTwoParentFingerprint : MethodFingerprint(
-    "L", AccessFlags.PUBLIC or AccessFlags.STATIC, listOf("L"),
-    strings = listOf("VIDEO_QUALITIES_QUICK_MENU_BOTTOM_SHEET_FRAGMENT")
+    "L", AccessFlags.PUBLIC or AccessFlags.STATIC, null, null,
+    strings = listOf("Callback already registered.", "Failed to create SpotlightModeController.")
```

Comment:
Does this also guarantee compatibility with the older versions?
```suggestion
    strings = listOf("Callback already registered.", "Failed to create SpotlightModeController.")
```